src/Wizard/Platform/Readline/ReadlinePlatform.ts

```diff
@@ -98,24 +106,24 @@ export class ReadlinePlatform implements Platform {
 
     await this.processConnectivity(TestType.HTTP);
 
-    await this.process('Verifying provided Token and Repeater ID', () =>
-      this.connectivityService.verifyAccess(TestType.AUTH)
-    );
+    if (!this.options.networkTestOnly) {
 
-    process.stdout.write(EOL);
+      await this.process('Verifying provided Token and Repeater ID', () =>
+        this.connectivityService.verifyAccess(TestType.AUTH)
+      );
+
+      process.stdout.write(EOL);
+    }
 
     console.log('EXTERNAL communication diagnostics completed.');
   }
 
   private async processInternalCommunication(): Promise<void> {
-    console.log(
-      `Next step is to validate the connection to your INTERNAL (local) target application(s).${EOL}`
-    );
-
+      console.log(
+        `Next step is to validate the connection to your INTERNAL (local) target application(s).${EOL}`
+      );
```

Comment:
Please make sure your git fires hooks properly. There are a lot of formatting issues here (e.g. indents, spaces, etc)